.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -296,14 +296,21 @@ jobs:
           awk '/^Failed Tests|Testing Time/{flag=1}/FAILED: CMakeFiles/{flag=0}flag' e2e.log >> $GITHUB_STEP_SUMMARY
         fi
         exit $exit_code
+    - name: List excluded SYCL CTS categories
+      if: inputs.tests_selector == 'cts'
+      run: |
+        cat $PWD/devops/cts_exclude_filter
     - name: Build SYCL CTS tests
       if: inputs.tests_selector == 'cts'
       env:
         CMAKE_EXTRA_ARGS: ${{ inputs.extra_cmake_args }}
       run: |
+        # Do not exclude any category if passing the CTS_TESTS_TO_BUILD variable.
+        cts_exclude_filter=$([ -n "$CTS_TESTS_TO_BUILD" ] && echo "" || echo "$PWD/devops/cts_exclude_filter")
+
```

Comment:
Can we move that to the "list" step with some renaming there. I think we can just update the env variable in place if we write the updated value to a proper `$GITHUB_SOMETHING`.

Reply:
Not sure what you mean. Could you provide a bit more info?

Reply:
I think we shouldn't print "excluded" categories if we ultimately won't exclude them.